README.md

```diff
@@ -133,8 +134,8 @@ An additional parameter to the Fetch API then allows the browser to include a si
 ```
 fetch(<resource-url>, {
   ...
-  signedRedemptionRecord: [issuer],
-  signRequestData: <include, omit, headers-only>,
+  signedRedemptionRecord: <issuer>,
+  signRequestData: include |/ omit | headers-only,
```

Comment:
I don't think you need the forward slash

Reply:
Indeed